providers/common/der/build.info

```diff
@@ -34,10 +34,9 @@ IF[{- !$disabled{ec} -}]
 
   DEPEND[${DER_EC_AUX/.c/.o}]=$DER_EC_H $DER_DIGESTS_H
   DEPEND[${DER_EC_GEN/.c/.o}]=$DER_EC_H
-ENDIF
+  GENERATE[$DER_EC_H]=$INCDIR/der_ec.h.in
+  DEPEND[$DER_EC_H]=oids_to_c.pm EC.asn1
```

Comment:
These two lines should be removed - looks like an incorrectly resolved conflict. Other than that this looks good now.

Reply:
And I only did PR #160 today :). Sigh.  fixup commit pushed. thanks.